tests/unit/components/sl-menu-test.js

```diff
@@ -7,7 +7,7 @@ var clickCounter = 0,
     modelStub = {
         label: null,
         pages: [
-            { label: 'Top Level A', classNames: [ 'class1', 'class2' ], pages: [
+            { label: 'Top Level A', className: 'class1 class2', pages: [
```

Comment:
Prefer that these be specified as an array rather than a text string.  It will be easier to dynamically manipulate an array of options passed to this component than a text string.